### DISCUSSION TITLE
Next js with CakePHP backend

### DISCUSSION THREAD

**Discussion by sadam1807** [vercel/next.js#23982] [Help]
@HaNdTriX 
Hi guys! 

Is it possible to create a website with the next js as front-end and PHP or CakePHP as backend with the same domain? 
For example

domain.com - Next website
domain.com/subfolder - PHP website ( /pages or /public folder?)
Thanks in advance

**Comment by akellbl4** [comment]
Hi @sadam1807,
Yes, you can. This problem has numerous different approaches to solve. But if you want to solve that only with Next.js you can use `rewrites` in `next.config.js` [there are docs](https://nextjs.org/docs/api-reference/next.config.js/rewrites). With `rewrites` you can proxy requests to PHP server.
Also, you can solve it with proxy servers like nginx, caddy, traefik, etc.

UPD: I wanted to add, It's not about where you store files. It's about how you run your applications and route requests between them.
